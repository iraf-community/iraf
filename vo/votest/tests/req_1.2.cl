#
#  Demonstrate row and column selection in a VOTable.

# Set the test description string.
votest.descr = "Demonstrate row and column selection in a VOTable"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"

print ("------------------------------------------------------------------")
print ("Req  1.2:  Users shall be able to select specific rows and/or")
print ("           columns of a VOTable using the existing task functionality.")
print ("------------------------------------------------------------------")


fcache init

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
tprint ("data$usno-b.xml", columns="sRa,sDec", rows="1-10", showhdr-)

print ("\nHTTP URI:  " // s1)				# remote http URI
tprint (s1, columns="sRa,sDec", rows="1-10", showhdr-)

print ("\nFile URI:  " // s2)				# file URI
tprint (s2, columns="sRa,sDec", rows="1-10", showhdr-)

print ("\nFile URI:  " // s3)				# file URI
tprint (s3, columns="sRa,sDec", rows="1-10", showhdr-)

