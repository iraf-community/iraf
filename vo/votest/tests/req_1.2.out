------------------------------------------------------------------
Req  1.2:  Users shall be able to select specific rows and/or
           columns of a VOTable using the existing task functionality.
------------------------------------------------------------------

Logical Path:  data$usno-b.xml
    1         999         999
    2          89          53
    3         999          16
    4         486         192
    5         405         569
    6         197         182
    7         173         234
    8         315         340
    9         614          65
   10         125         100

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
    1         999         999
    2          89          53
    3         999          16
    4         486         192
    5         405         569
    6         197         182
    7         173         234
    8         315         340
    9         614          65
   10         125         100

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
    1         999         999
    2          89          53
    3         999          16
    4         486         192
    5         405         569
    6         197         182
    7         173         234
    8         315         340
    9         614          65
   10         125         100

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
    1         999         999
    2          89          53
    3         999          16
    4         486         192
    5         405         569
    6         197         182
    7         173         234
    8         315         340
    9         614          65
   10         125         100
